src/storage/v2/disk/storage.cpp

```diff
@@ -496,9 +538,27 @@ VerticesIterable DiskStorage::DiskAccessor::Vertices(LabelId label, PropertyId p
   index_deltas_storage_.emplace_back(std::list<Delta>());
```

Comment:
I think you can pass here also empty to avoid creating temporary:
`index_deltas_storage_.emplace_back();`